### PR TITLE
Wrap SBCK

### LINKED ISSUE #1109: Add experimental wrapping of SBCK to sdba
### Generic Issue

Yoan Robin's [SBCK](https://github.com/yrobink/SBCK) implements many bias adjustment methods, including most interesting multivariate ones. It follows a fit-predict syntax quite similar to xclim's sdba. It uses numpy, but only acts (AFAIU) on timeseries (by that I mean that is doesn't try to vectorize or loop the adjustment, there's no parallelism).

Which means, it could be quite easy (I usually underestimate things) to wrap it up into xclim. I'm talking of automatically generating sdba Adjustment classes from SBCK ones, and using `apply_ufunc` to loop on elements of the xarray inputs (like gridpoints. This would allow usage of dask with the same constraints as the other methods.

However, the license (GPL3) doesn't allow sub-licensing, which means we can't copy the code as-is into xclim. Let's just hope there's no bug...

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by aulemahal**
### Pull Request Checklist:
- [x] This PR addresses an already opened issue (for bug fixes / features)
    - This PR fixes #1109 
- [x] Tests for the changes have been added (for bug fixes / features)
  - [x] (If applicable) Documentation has been added / updated (for bug fixes / features)
- [x] HISTORY.rst has been updated (with summary of main changes)
  - [x] Link to issue (:issue:`number`) and pull request (:pull:`number`) has been added
- [x] The relevant author information has been added to `.zenodo.json`

### What kind of change does this PR introduce?

* All adjustment methods of [SBCK](https://github.com/yrobink/SBCK) are wrapped in `sdba.adjustment`, using `Adjust` classes (single step mode). This allow the use of dask and xarray, but is quite basic otherwise.

### Does this PR introduce a breaking change?
No.

### Other information:
SBCK depends on `Eigen` a C++ library. It is available on conda, but the CI uses pip, so I can't install it to run the new test. Or am I wrong?

### CODE REVIEW

**Comment by Zeitsperre** [comment]
@aulemahal Not sure why, but the test is being skipped.
```
xclim/testing/tests/test_sdba/test_adjustment.py::test_SBCK[True-method0] SKIPPED [ 94%]
```
I'll leave you to do the configuring there.

**Comment by aulemahal** [comment]
haha, sorry to tell you that, @Zeitsperre , but the test is skipped because SBCK isn't being installed ;)
The last commit should have fixed the issue, it seems related with tox's naming tricks.

**Comment by Zeitsperre** [comment]
That's interesting. You're right. I feel like I've been able to throw keywords at the end of builds (e.g. `-lm3`) in order to trigger specific action in the main `testenv` block, but adding `-slow` broke that one. I wonder if it was a 'bug'/'feature'?

**Comment by aulemahal** [comment]
My guess is that the env name needs to be a bit more specific. All other keyword-based variation we have happen in the generic unnamed env. May be `sbck-{,slow}` ? The doc has examples with choices (like `django{15,16}`) but not with an optional flag.

File: xclim/sdba/__init__.py [hunk 1]
```diff
@@ -70,18 +83,19 @@
...
     stack_variables,
     unpack_moving_yearly_window,
     unstack_variables,
 )
-from .properties import *
 
 # TODO: ISIMIP ? Used for precip freq adjustment in biasCorrection.R
 # Hempel, S., Frieler, K., Warszawski, L., Schewe, J., & Piontek, F. (2013). A trend-preserving bias correction &ndash;
 # The ISI-MIP approach. Earth System Dynamics, 4(2), 219–236. https://doi.org/10.5194/esd-4-219-2013
+if hasattr(adjustment, "_generate_SBCK_classes"):
```
Comment (huard) [inline xclim/sdba/__init__.py:99]:
```suggestion
# If SBCK is installed, create adjustment classes wrapping SBCK's algorithms.
if hasattr(adjustment, "_generate_SBCK_classes"):
```

**Pull request merged**